pkg/ipamd/datastore/data_store.go

```diff
@@ -122,6 +122,19 @@ var (
 		},
 		[]string{"cidr"},
 	)
+	noAvailableIPAddrs = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "awscni_err_no_avail_addrs",
+			Help: "The number of pod IP assignments that fail due to no available IP addresses",
+		},
+	)
+	eniUtilization = prometheus.NewGaugeVec(
```

Comment:
sorry, I should have pointed this out earlier. Can we rename this to something like `eniIPsInUse`?